### PR TITLE
add section to enable remote visu via vnc

### PR DESCRIPTION
**Pull request by sandrineBeauche**

### CODE REVIEW

File: src/docs/admin/ProActiveAdminGuide.adoc [hunk 1]
```diff
@@ -805,6 +805,22 @@ HTTPS can be enabled for Web applications using a Java keystore. See `web.https`
 for the possible settings. We provided a default self-signed keystore than can be used for testing but for
 production you should build your own keystore with a valid certificate so it will be accepted by browsers.
 
+=== Enable VNC remote visualization in the browser
+
+The Rest API web application embeds a proxy to allow the remote display visualization of a ProActive Node running a given task
+via a VNC server from the browser. To enable this feature, the scheduler has to configured:
```
Comment (lpellegr) [inline src/docs/admin/ProActiveAdminGuide.adoc]: "has to **be** configured" or "requires configuration"?

File: src/docs/admin/ProActiveAdminGuide.adoc [hunk 2]
```diff
@@ -805,6 +805,22 @@ HTTPS can be enabled for Web applications using a Java keystore. See `web.https`
...
+
+* configure the proxy: edit the `PROACTIVE_HOME/config/web/settings.ini` file and set `novnc.enabled` to true in order
+to start the proxy.
+* configure the scheduler portal to use the proxy when opening a new tab browser that show the remote visualization:
```
Comment (lpellegr) [inline src/docs/admin/ProActiveAdminGuide.adoc]: "that show**s**"

File: src/docs/admin/ProActiveAdminGuide.adoc [hunk 3]
```diff
@@ -805,6 +805,22 @@ HTTPS can be enabled for Web applications using a Java keystore. See `web.https`
...
+edit the `PROACTIVE_HOME/dist/war/scheduler/scheduler.conf`.
+** Set `sched.novnc.url` to the public address of the proxy. This public address is the public address of the host where the sheduler is started,
+and the port specified by the option `novnc.port` in the file `PROACTIVE_HOME/config/web/settings.ini`
+** Set `sched.novnc.page.url` to the public address of the vnc client to be executed in the browser. The client is located
```
Comment (lpellegr) [inline src/docs/admin/ProActiveAdminGuide.adoc]: VNC in uppercase?

File: src/docs/admin/ProActiveAdminGuide.adoc [hunk 4]
```diff
@@ -805,6 +805,22 @@ HTTPS can be enabled for Web applications using a Java keystore. See `web.https`
...
+in the REST API with the file `novnc.html`.
+
+For more informations, see  https://github.com/ow2-proactive/scheduling/blob/master/rest/README-novnc.txt
```
Comment (lpellegr) [inline src/docs/admin/ProActiveAdminGuide.adoc]: Remove one space before URL?

File: src/docs/admin/ProActiveAdminGuide.adoc (hunk 1 above)
Comment (marcocast) [inline src/docs/admin/ProActiveAdminGuide.adoc]: has to be configured

File: src/docs/admin/ProActiveAdminGuide.adoc [hunk 5]
```diff
@@ -805,6 +805,22 @@ HTTPS can be enabled for Web applications using a Java keystore. See `web.https`
...
+
+The Rest API web application embeds a proxy to allow the remote display visualization of a ProActive Node running a given task
+via a VNC server from the browser. To enable this feature, the scheduler has to configured:
+
+* configure the proxy: edit the `PROACTIVE_HOME/config/web/settings.ini` file and set `novnc.enabled` to true in order
+to start the proxy.
+* configure the scheduler portal to use the proxy when opening a new tab browser that show the remote visualization:
+edit the `PROACTIVE_HOME/dist/war/scheduler/scheduler.conf`.
+** Set `sched.novnc.url` to the public address of the proxy. This public address is the public address of the host where the sheduler is started,
+and the port specified by the option `novnc.port` in the file `PROACTIVE_HOME/config/web/settings.ini`
```
Comment (marcocast) [inline src/docs/admin/ProActiveAdminGuide.adoc]: no need to put , before "and"

File: src/docs/admin/ProActiveAdminGuide.adoc (hunk 4 above)
Comment (marcocast) [inline src/docs/admin/ProActiveAdminGuide.adoc]: information without s

**Comment by marcocast** [comment]
few grammar typos, for the rest is ok for me

File: src/docs/admin/ProActiveAdminGuide.adoc (hunk 1 above)
Comment (paraita) [inline src/docs/admin/ProActiveAdminGuide.adoc]: done

File: src/docs/admin/ProActiveAdminGuide.adoc (hunk 2 above)
Comment (paraita) [inline src/docs/admin/ProActiveAdminGuide.adoc]: done

File: src/docs/admin/ProActiveAdminGuide.adoc (hunk 3 above)
Comment (paraita) [inline src/docs/admin/ProActiveAdminGuide.adoc]: done

**Pull request merged**